Dockerfiles/manifests/agent-only/README.md

```diff
@@ -1,9 +1,10 @@
 The kubernetes manifests found in this directory have been automatically generated
 from the [helm chart `datadog/datadog`](https://github.com/DataDog/helm-charts/tree/master/charts/datadog)
-version 3.11.0 with the following `values.yaml`:
+version 3.33.6 with the following `values.yaml`:
 
 ```yaml
 datadog:
+  apiKey: <DATADOG_API_KEY>
```

Comment:
should we also add `appKey` ?

Reply:
I don't think so, because unlike apiKey, appKey does not appear in an `if` that decides whether a manifest should be generated.